packages/@aws-cdk/aws-ec2/package.json

```diff
@@ -231,6 +231,7 @@
       "docs-public-apis:@aws-cdk/aws-ec2.Subnet.subnetNetworkAclAssociationId",
       "docs-public-apis:@aws-cdk/aws-ec2.Subnet.subnetVpcId",
       "docs-public-apis:@aws-cdk/aws-ec2.Subnet.fromSubnetAttributes",
+      "docs-public-apis:@aws-cdk/aws-ec2.Subnet.fromSubnetId",
```

Comment:
Don't ignore the linter warning, fix the warning by adding a documentation string!